<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QHMainWindow</class>
 <widget class="QMainWindow" name="QHMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QMdiArea" name="mdiArea">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>541</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actDocNew"/>
   <addaction name="actDocOpen"/>
   <addaction name="actViewMode"/>
   <addaction name="actCascade"/>
   <addaction name="actTile"/>
   <addaction name="actCloseALL"/>
  </widget>
  <action name="actDocNew">
   <property name="text">
    <string>新建</string>
   </property>
  </action>
  <action name="actDocOpen">
   <property name="text">
    <string>打开文档</string>
   </property>
  </action>
  <action name="actCascade">
   <property name="text">
    <string>级联展开</string>
   </property>
  </action>
  <action name="actTile">
   <property name="text">
    <string>平铺展开</string>
   </property>
  </action>
  <action name="actCloseALL">
   <property name="text">
    <string>关闭所有子窗口</string>
   </property>
  </action>
  <action name="actViewMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>MDI模式</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
